test/cmdlineTests.sh

```diff
@@ -240,35 +240,42 @@ printTask "Running general commandline tests..."
     cd "$REPO_ROOT"/test/cmdlineTests/
     for tdir in */
     do
-        if [ -e "${tdir}/input.json" ]
+        printTask " - ${tdir}"
+
+        inputFiles="$(find "${tdir}" -name 'input.*' -type f -exec printf "%s\n" "{}" \;)"
+        inputCount="$(echo "${inputFiles}" | wc -l)"
+        if (( ${inputCount} == 0 ))
```

Comment:
```suggestion
        if (( inputCount == 0 ))
```
I _think_ this works inside parens?

Reply:
Oh, it does. Not sure I like it though :) Now I'll be second guessing every time if it's really a variable or just a string.

Reply:
Ok, fine with keeping like this.